TaskLayer/GPTMDTask/GPTMDParameters.cs

```diff
@@ -5,13 +5,16 @@ namespace EngineLayer
 {
     public class GptmdParameters
     {
+        /// <summary>
+        /// The default modifications for a G-PTM-D search can be altered here by adding to b.Modification.Type.Equals("  NAME OF MODS TO ADD TO DEFAULT ")
+        /// </summary>
         public GptmdParameters()
         {
             ListOfModsGptmd = GlobalVariables.AllModsKnown.Where(b =>
                 b.ModificationType.Equals("Common Artifact") ||
-                b.ModificationType.Equals("Common Biological") ||
-                b.ModificationType.Equals("Metal") ||
-                b.ModificationType.Equals("Less Common")
+                b.ModificationType.Equals("Common Biological")
+                //|| b.ModificationType.Equals("Metal") 
+                //|| b.ModificationType.Equals("Less Common")
```

Comment:
My main concern was the less common mods. Have you been unchecking metals, too?

Reply:
i do. This change may go away. I plan to ask many users about this change before it goes through.

Reply:
Sounds good 👍

Reply:
I personally keep metals in my GPTMD search, I feel that (at least some of them) go along with common artifacts.

Reply:
I kind of believe the metal adducts, too...

Reply:
Have we ever done a group FDR for these different mod groups?

Reply:
not per se. But, using TMT as a fixed mod, which I think would be the general process, means that everything is the group..... I guess.